walker/walk.go

```diff
@@ -19,6 +20,7 @@ type WalkFunc func(filePath string, info os.FileInfo, opener analyzer.Opener) er
 
 func isIgnored(filePath string) bool {
 	filePath = strings.TrimLeft(filePath, "/")
+	filePath = filepath.Clean(filePath)
```

Comment:
What about inserting `filepath.ToSlash` before L22? In that case, we probably need to replace `utils.PathSeparator` with `/`.

```
	filePath = filepath.ToSlash(filePath)
        filePath = strings.TrimLeft(filePath, "/")
	filePath = filepath.Clean(filePath)
```